gradle/libs.versions.toml

```diff
@@ -63,6 +63,7 @@ hilt-work = "1.2.0"
 # Android UI
 accompanist = "0.32.0" # adjusted to work with compose-destinations "1.9.54"
 material = "1.12.0"
+material3 = "1.3.0-beta05"
```

Comment:
there is already [1.3.0-rc01](https://developer.android.com/jetpack/androidx/releases/compose-material3#1.3.0-rc01) available maybe we can give it a try